pyke/lightcurve.py

```diff
@@ -143,6 +171,7 @@ def __init__(self, time, flux, flux_err=None, centroid_col=None,
         self.quarter = quarter
         self.mission = mission
         self.cadenceno = cadenceno
+        self.id = id
```

Comment:
Let's use ``keplerid`` here in order to keep consistency with the same property in ``KeplerTargetPixelFile``

Reply:
👍